test/e2e/states.go

```diff
@@ -135,3 +135,14 @@ func ifaceDownIPv4Disabled(iface string) nmstatev1alpha1.State {
         enabled: false
 `, iface))
 }
+
+func ifaceUpWithDynamicIP(infName string) nmstatev1alpha1.State {
+	return nmstatev1alpha1.NewState(fmt.Sprintf(`interfaces:
+    - name: %s
+      type: vlan
+      state: up
+      ipv4:
+        dhcp: true
```

Comment:
This will always fail since we don't have a DHCP server running on this VLAN segment. I guess we can make an exception in this case and test it with static IP (while there is DHCP in tests). I want to get this in, without investing in configuration of VLAN DHCP server.

Reply:
Strange, I thought it just set the dchp option to auto, not actually wait for it to get dhcp

Reply:
Also any idea how a policy for setting static IP  on all nodes looks like?

Reply:
It does wait and fails if no IP is obtained (or at least it should)